### PR TITLE
Updated wording for long oc install prereqs

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
Related to #25227 - updating instances of old wording of installing the CLI.

**Comment by jboxman** [comment]
Mostly just me.

File: modules/virt-creating-blank-disk-datavolumes.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ customizing and deploying a DataVolume configuration file.
 .Prerequisites
 
 * At least one available PersistentVolume
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`
+* Install the OpenShift CLI (`oc`)
```
Comment (bobfuru) [inline modules/virt-creating-blank-disk-datavolumes.adoc:14]: Missing a period.

File: modules/machine-delete.adoc [hunk 2]
```diff
@@ -10,7 +10,7 @@ You can delete a specific machine.
 .Prerequisites
 
 * Install an {product-title} cluster.
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`
+* Install the OpenShift CLI (`oc`)
```
Comment (lbarbeevargas) [inline modules/machine-delete.adoc:13]: Missing period at end.

**Review by bobfuru — COMMENTED** [review]
One suggestion, and otherwise LGTM!

File: modules/machineset-creating.adoc [hunk 3]
```diff
@@ -17,7 +17,7 @@ specific workloads of your choice.
 .Prerequisites
 
 * Deploy an {product-title} cluster.
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`
+* Install the OpenShift CLI (`oc`)
```
Comment (lbarbeevargas) [inline modules/machineset-creating.adoc:20]: Missing period at end.

File: modules/rhel-adding-more-nodes.adoc [hunk 4]
```diff
@@ -24,7 +24,7 @@ used to install the cluster are on the machine that you use the run the playbook
 access to all of the RHEL hosts.
 * If you use SSH key-based authentication, you must manage the key with an
 SSH agent.
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`,
+* Install the OpenShift CLI (`oc`),
```
Comment (lbarbeevargas) [inline modules/rhel-adding-more-nodes.adoc:27]: Not sure there needs to be a comma after (`oc`).

File: modules/rhel-preparing-playbook-machine.adoc [hunk 5]
```diff
@@ -13,7 +13,7 @@ cluster but must be able to access it.
 
 .Prerequisites
 
-* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`,
+* Install the OpenShift CLI (`oc`),
```
Comment (lbarbeevargas) [inline modules/rhel-preparing-playbook-machine.adoc:16]: Not sure there needs to be a comma after (`oc`).

File: modules/virt-creating-blank-disk-datavolumes.adoc (hunk 1 above)
Comment (lbarbeevargas) [inline modules/virt-creating-blank-disk-datavolumes.adoc:14]: Missing period at end.

**Review by lbarbeevargas — COMMENTED** [review]
Just a couple of comments and missing periods. Otherwise, LGTM.

**Pull request merged**